package.json

```diff
@@ -13,6 +13,9 @@
   "engines": {
     "node": ">=8"
   },
+  "resolutions": {
+    "handlebars": "4.4.5"
+  },
```

Comment:
Using `resolutions` is always bad idea, it helps in applications to broken/vuln dependency releases, but not in libraries or tools